### PR TITLE
Feature/invite friends

### PR DESCRIPTION
**Pull request by sandrohp88**
+ Add reusable components Modal and Chip.
+ Translate mockup to code
+ Create a new Component InviteFriends

May need some styles improvements.
I'll gladly do any requested change or improvement  

[Trello card](https://trello.com/c/5buDJdzQ)

image

### CODE REVIEW

**Comment by ArnasDickus** [comment]
@sandrohp88  Usually I don't fix merge conflicts. However, it was a simple main.js conflict. So fixed it for you. Here's [changes](https://github.com/zero-to-mastery/mappypals/pull/124/commits/4594c07f5b763c79889b1d62ca74be6e5b5a45d1)

File: src/components/UI/Modal/Modal.module.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+.closeButton {
+    border: none;
+    background: none;
+    width: auto;
+    margin-top: 16px;
+    margin-right: 16px;
+    font-size: 18px;
+    text-align: end;
+    &:hover {
+        font-size: 20px;
```
Comment (ArnasDickus) [inline src/components/UI/Modal/Modal.module.scss]: on hover increasing font-size. Slightly push down Send an invite to multiple friends. I suggest removing this.

File: src/components/UI/Modal/Modal.module.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100vw;
+    min-width: 50%;
+    z-index: 2;
+}
+
+.modalMain {
+    margin-top: 50%;
```
Comment (ArnasDickus) [inline src/components/UI/Modal/Modal.module.scss]: I suggest using the position absolute to center the application. Similarly how it's done [here](https://github.com/zero-to-mastery/mappypals/blob/master/src/pages/Home/AddFriendForm/InvitationSent.css)

File: src/pages/InviteFriends/InviteFriends.module.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+.emailList {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: left;
+}
+.sendInvite{
+    margin:16px auto;
+}
```
Comment (ArnasDickus) [inline src/pages/InviteFriends/InviteFriends.module.scss]: Needs a new line here. Try running `npm run format`

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding PR. I finally got around to review it. So here it goes:

1) Adding too many emails would break application styling.
2) Application is not mobile-friendly.
on-mobile-not-responsive
3) Clicking on any other navbar links should remove. Send an invite form. I think your best bet is to use redux, for that.
4) Invite form should be in the center
5) Also, it would be great if only one form should be active at the time. This thing is used in map.js render. Using if statements.

**Comment by mubarakshow** [comment]
@sandrohp88 in addition to @ArnasDickus's review: 
The form shouldn't allow the same email to be added more than once.

File: src/pages/InviteFriends/InviteFriends.module.scss (hunk 3 above)
Comment (sandrohp88) [reply src/pages/InviteFriends/InviteFriends.module.scss]: @ArnasDickus @mubarakshow . Thanks for the comments. I'll try to fix those issues by today

File: src/components/UI/Modal/Modal.module.scss (hunk 1 above)
Comment (sandrohp88) [reply src/components/UI/Modal/Modal.module.scss]: @ArnasDickus Regarding to have just one active modal form at the time, to make that possible probably I'll have to use redux (global state). That way I  can also hide this form when clicking on any other navbar link

**Comment by ArnasDickus** [comment]
Alright, everything works fine.

**Review by ArnasDickus — APPROVED** [review]

**Pull request merged**